modules/rosa-policy-incident.adoc

```diff
@@ -102,20 +102,22 @@ Platform audit logs are securely forwarded to a centralized security information
 
 [id="rosa-policy-incident-management_{context}"]
 == Incident management
-An incident is an event that results in a degradation or outage of one or more Red{nbsp}Hat services. An incident can be raised by a customer or a Customer Experience and Engagement (CEE) member through a support case, directly by the centralized monitoring and alerting system, or directly by a member of the SRE team.
+An incident is an event that results in a degradation or outage of one or more Red{nbsp}Hat services, and can affect Service Level Agreements (SLAs). An incident can be raised by a customer or a Customer Experience and Engagement (CEE) member through a support case, directly by the centralized monitoring and alerting system, or directly by a member of the SRE team.
 
 Depending on the impact on the service and customer, the incident is categorized in terms of link:https://access.redhat.com/support/offerings/production/sla[severity].
 
+Red Hat either sends out cluster notifications to affected individual clusters or changes the status at link:https://status.redhat.com[status.redhat.com] to reflect a wider incident. Cluster notifications are not sent for low-impact events, low-risk security updates, routine operations and maintenance, or minor, transient issues that are quickly resolved by SRE.
+
 When managing a new incident, Red{nbsp}Hat uses the following general workflow:
 
 . An SRE first responder is alerted to a new incident and begins an initial investigation.
 . After the initial investigation, the incident is assigned an incident lead, who coordinates the recovery efforts.
-. An incident lead manages all communication and coordination around recovery, including any relevant notifications and support case updates.
+. An incident lead manages all communication and coordination around recovery, including any relevant notifications and support case updates. If the status of a service changes or if Red Hat has a significant update on the progress, then the incident lead sends out an updated cluster notification.
```

Comment:
I keep thinking we ought to make an attribute for `Red{nbsp}Hat` so no one ever has to think about this again :sweat_smile: 
```suggestion
. An incident lead manages all communication and coordination around recovery, including any relevant notifications and support case updates. If the status of a service changes or if Red{nbsp}Hat has a significant update on the progress, then the incident lead sends out an updated cluster notification.
```